girder/api/rest.py

```diff
@@ -1087,9 +1087,14 @@ def getPagingParameters(self, params, defaultSortField=None, defaultSortDir=Sort
         :param defaultSortDir: Sort direction.
         :type defaultSortDir: girder.constants.SortDir
         """
-        offset = int(params.get('offset', 0))
-        limit = int(params.get('limit', 50))
-        sortdir = int(params.get('sortdir', defaultSortDir))
+        try:
+            offset = int(params.get('offset', 0))
+            limit = int(params.get('limit', 50))
+            sortdir = int(params.get('sortdir', defaultSortDir))
+        except ValueError:
+            raise RestException('Invalid value for offset, limit, or sortdir parameter.')
+        if sortdir not in [-1, 1]:
```

Comment:
This should use the values defined in `SortDir`:  https://github.com/girder/girder/blob/49e2bdbc387212778940abc10fc4ed4508d492aa/girder/constants.py#L224